src/axom/mir/docs/sphinx/mir_utilities.rst

```diff
@@ -0,0 +1,235 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+MIR Blueprint Utilities
+******************************************************
+
+The MIR component contains several useful building blocks for writing algorithms
+for Blueprint meshes.
+
+ * Structured as classes with an ``execute()`` method
+ * Often templated on execution space and views
+
+#######################
+Copying Blueprint Data
+#######################
+
+If a ``conduit::Node`` containing Blueprint data is not on the desired memory space, it can be
+moved using the ``axom::mir::utilities::blueprint::copy<ExecSpace>()`` function. The ``copy``
+function copies the source ``conduit::Node`` to the destination ``conduit::Node``, making sure
+to use the appropriate Axom allocator for non-string bulk arrays (e.g. arrays of ints, floats,
+doubles, etc.). Data small enough to fit in a ``conduit::Node`` and strings are left in the host
```

Comment:
Add sentence:  "The template argument to `copy` specifies the destination data space."  (See src/axom/mir/utilities/blueprint_utilities.hpp line 250.)

Reply:
Done